modules/Bio/EnsEMBL/Analysis/Hive/Config/production_rnaseq_db.pm

```diff
@@ -0,0 +1,663 @@
+
+    # Please assign some or all columns from the summary file to the
+    # some or all of the following categories.  Multiple values can be
+    # separted with commas. ID, SM, DS, CN, is_paired, filename, read_length, is_13plus,
+    # is_mate_1 are required. If pairing_regex can work for you, set is_mate_1 to -1.
+    # You can use any other tag specified in the SAM specification:
+    # http://samtools.github.io/hts-specs/SAMv1.pdf
+
+    ####################################################################
+    # This is just an example based on the file snippet shown below.  It
+    # will vary depending on how your data looks.
+    ####################################################################
+    file_columns      => ['SM', 'ID', 'is_paired', 'filename', 'is_mate_1', 'read_length', 'is_13plus', 'CN', 'PL', 'DS'],
+    long_read_columns => ['sample','filename'],
+
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+########################
+# db info
+########################
+    'rnaseq_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_rnaseq_'.$self->o('release_number'),
+      -host   => $self->o('rnaseq_db_server'),
+      -port   => $self->o('rnaseq_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_blast_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_rnaseq_blast_'.$self->o('release_number'),
+      -host   => $self->o('rnaseq_blast_db_server'),
+      -port   => $self->o('rnaseq_blast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_refine_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_refine_'.$self->o('release_number'),
+      -host   => $self->o('rnaseq_refine_db_server'),
+      -port   => $self->o('rnaseq_refine_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'production_db' => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ensembl_production',
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+
+    return [
+    # inheriting database and hive tables' creation
+      @{$self->SUPER::pipeline_create_commands},
+
+      'mkdir -p '.$self->o('rnaseq_dir'),
+    ];
+}
+
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+
+    # TODO
+    # the following isn't used in this subpipeline, is it needed?
+    skip_rnaseq => $self->o('skip_rnaseq'),
+  }
+}
```

Comment:
This is not needed we will not skip any pipeline

Reply:
Does this subroutine load the pipeline wide parameters or not needed at all?

Reply:
Yes it does but it is not needed in this case as we don't need pipeline wide parameters.